src/observer.hoc.ts

```diff
@@ -12,10 +12,10 @@ import { global } from "./Observable";
 
 function useObservable<T>(fn: () => T, name: string) {
     const { 0: value, 1: render } = useState(0)
-    const cb = () => render(value + 1)
+    const cb = () => render((prev) => prev + 1)
```

Comment:
```suggestion
    const cb = () => render((prev) => 1 - prev)
```

will flip-flop forever and never overflow or lose precision